test/smoke/Makefile

```
@@ -27,3 +27,6 @@ external-dns:
 
 multus-cni:
 	cd "${ROOT_DIR}"/test/e2e && ginkgo -v -- --packages="multus-cni" --version="3.7.1" --guest-cluster-name="tce.public"
+
+gatekeeper:
+	cd "${ROOT_DIR}"/test/gatekeeper && ./e2e-test.sh && tanzu package installed delete gatekeeper -y
```

I think we can avoid using `cd` here. Something like this

```suggestion
	./"${ROOT_DIR}"/test/gatekeeper/e2e-test.sh && tanzu package installed delete gatekeeper -y
```